pkg/polaris/ctl_polaris.go

```diff
@@ -124,13 +126,12 @@ func (ctl *CRSpecBuilderFromCobraFlags) AddCRSpecFlagsToCommand(cmd *cobra.Comma
 	cmd.Flags().StringVar(&ctl.Registry, "registry", ctl.Registry, "Docker registry e.g. docker.io/myuser")
 
 	// Those flags need be hidden since external databases are not currently supported
-	cmd.Flags().StringVar(&ctl.PostgresHost, "postgres-host", ctl.PostgresHost, "")
-	cmd.Flags().IntVar(&ctl.PostgresPort, "postgres-port", ctl.PostgresPort, "")
-	cmd.Flags().MarkHidden("postgres-host")
-	cmd.Flags().MarkHidden("postgres-port")
-
+	cmd.Flags().StringVar(&ctl.PostgresHost, "postgres-host", GetPolarisDefault().PolarisDBSpec.PostgresDetails.Host, "Postgres Host")
+	cmd.Flags().IntVar(&ctl.PostgresPort, "postgres-port", GetPolarisDefault().PolarisDBSpec.PostgresDetails.Port, "PostgresPort")
 	cmd.Flags().StringVar(&ctl.PostgresUsername, "postgres-username", ctl.PostgresUsername, "Postgres username")
 	cmd.Flags().StringVar(&ctl.PostgresPassword, "postgres-password", ctl.PostgresPassword, "Postgres password")
+	cmd.Flags().StringVar(&ctl.PostgresSSLMode, "postgres-ssl-mode", string(GetPolarisDefault().PolarisDBSpec.PostgresDetails.SSLMode), "Postgres ssl mode [disable|require]")
+	cmd.Flags().BoolVar(&ctl.PostgresInternal, "postgres-container", GetPolarisDefault().PolarisDBSpec.PostgresDetails.IsInternal, "Not recommended. Use a PostgreSQL container")
```

Comment:
Sorry to nit, but this is still a bit confusing to me.  Maybe something like this would help
```
Use in-cluster postgres in a container (Not recommended)
```

Also, I'm still a little unclear on the interface we want to give here.  This is an if else workflow here.  If we are configuring in-cluster postgres here, should a customer care about the configuration.  Also, `--postgres-size` should not be set if using external postgres and `--postgres-ssl-mode` should not be set if using internal postgres, right?

Reply:
For in cluster postgres container, ssl mode is disabled that is the default value as well..